server/routes/api.js

```diff
@@ -17,15 +17,71 @@ router.route('/')
  * =============================================================
  */
 
+//insert entry into post table, return created entry
 router.route('/posts/new')
   .post(controller.Posts.createPost);
 
 router.route('/posts/user/:id')
   .get(controller.Posts.getPostsByUserId);
+  
+//return posts entries from post table by story id SORTED BY TIME STAMP
+router.route('/posts/story/:id')
+  .get(controller.Posts.getPostsByStoryId);
```

Comment:
I am going to need to pick your brain on how these work

Reply:
yeah pick my brain hole